lib/geoengineer/cli/geo_cli.rb

```diff
@@ -77,6 +83,22 @@ def require_from_pwd(file)
     require "#{Dir.pwd}/#{file}"
   end
 
+  def gps
+    return @gps if @gps
+    require_gps
+    @gps ||= GeoEngineer::GPS.parse_dir("#{Dir.pwd}/gps/")
+  end
+
+  def require_gps
```

Comment:
Ideally, we don't have to migrate all files to a specific `gps` directory, but can have `.yml` files live alongside `.rb` files, and `geo` is smart enough to load them based on file type

Reply:
GPS requires two features: 
1. it loads all GPS files every plan/apply to validate all nodes are correctly configured
2. its location corresponds 1-1 with the project name.

To get these it is much easier putting the yml files in their own directory. 

We could parse the projects directory, as well for yml files. For this first version I would like to keep them separate until some of the dust settles on other features.

Reply:
I think its also worthwhile to consider the cost of moving files around a lot and its toll on end users

Reply:
I think the best idea is to have a custom extension for gps files. That way they can live along side the original file as well.